tf/utils.go

```diff
@@ -136,6 +138,136 @@ func NewPassReport(plan *tfjson.Plan) types.PassReport {
 	return out
 }
 
+func NewCoverageReportFromState(state *tfjson.State) (types.CoverageReport, error) {
```

Comment:
>NewCoverageReportFromState [](http://example.com/codeflow?start=5&length=26)

could it be moved into `coverage` package?

Reply:
I think it's better to keep it for other function like `NewDiffReport` , `NewPassReportFromState` are in this file.